### PR TITLE
added collector script

### PR DESCRIPTION
**Pull request by Dzhango**

### CODE REVIEW

File: source/stats-page/stats.html [hunk 1]
```diff
@@ -16,92 +17,86 @@
...
+            border: none;
+            font-size: 2rem;
+            text-align: center;
+            font-weight: lighter;
+            border-radius: 1rem;
         }
         
-        50% {
-          transform: scale(1.4);
+        button:hover {
```
Comment (RainbowPangolin) [inline source/stats-page/stats.html:34]: Can you clarify which of these changes are the result of your linter and which are intentional? We should avoid making any unnecessary changes to the HTML right now since our code and tests are extremely brittle and rely on the tags.

**Review by RainbowPangolin — DISMISSED** [review]

File: source/stats-page/stats.html (hunk 1 above)
Comment (Dzhango) [reply source/stats-page/stats.html:34]: Oh shoot good catch. I am gonna redo the pull request.

**Review by cmwall0605 — APPROVED** [review]
